addons/godot-xr-tools/staging/staging.tscn

```diff
@@ -29,9 +31,27 @@ material/0 = SubResource( 3 )
 environment = SubResource( 2 )
 
 [node name="LoadingScreen" parent="." instance=ExtResource( 3 )]
-splash_screen = null
 progress = 0.0
 
 [node name="Scene" type="Spatial" parent="."]
 
 [node name="Tween" type="Tween" parent="."]
+
+[node name="ARVROrigin" type="ARVROrigin" parent="."]
```

Comment:
I wonder if we should rename the nodes to `XROrigin` and `XRCamera`, might make things more recognizable between 3 and 4? Or am I over thinking it?

Reply:
It has benefits for compatibility, but it may also be a source of confusion as the developers will still have to use the ARVRxxxx names in code. If we choose to do this then I would probably make it a separate PR as we'd want to sweep through other files and comments.